R/susie_wrapper.R

```diff
@@ -245,7 +245,8 @@ rss_input_preprocess = function(sumstats, LD_data, skip_region = NULL) {
 #' @import tibble
 #' @export
 susie_rss_pipeline = function(sumstat, R, ref_panel, n, L, var_y, QC = TRUE, impute = TRUE, bayesian_conditional_analysis = TRUE, rcond = 0.01, R2_threshold = 0.6,
-                              max_L = 20, l_step = 5, minimum_ld = 5) {
+                              max_L = 20, l_step = 5, minimum_ld = 5, coverage = 0.95,
+                              secondary_coverage = c(0.7, 0.5), signal_cutoff = 0.025) {
```

Comment:
@SamuelSun2000 here let's use default 0.1 still for the software. However, in the pipeline let's put it 0.025 which reflects my personal aggressiveness of our particular project ...